docs/reference/docfx-cli-reference.md

```diff
@@ -0,0 +1,166 @@
+---
+uid: docfx_cli_reference
+title: docfx CLI reference
+---
+`docfx` CLI Reference
+=========================
+
+## Introduction
+
+`docfx` is used to generate documentation for programs. It has the ability to:
+
+1. Extract language metadata for programing languages as defined in [Metadata Format Specification](../spec/metadata_format_spec.md). Currently `C#`, `VB` and `F#` are supported. The language metadata is saved in `YAML` format as described in [YAML 1.2][1].
```

Comment:
I'll leave people more expert in Docfx than me to comment but I can say that in the `Microsoft.DocAsCode.Dotnet` assembly there is support for compiling VB code to extract the metadata.  The class  `ExtractMetadataWorker` contains the following:

```c#
        if (_files.TryGetValue(FileType.VBSourceCode, out var vbFiles))
        {
            var compilation = CompilationHelper.CreateCompilationFromVBFiles(vbFiles.Select(f => f.NormalizedPath));
            hasCompilationError |= compilation.CheckDiagnostics(_config.AllowCompilationErrors);
            assemblies.Add((compilation.Assembly, compilation));
        }

```

Reply:
Input wise we support assemblies, `csproj`, `vbproj`, loose C# files and loose VB files. Output wise only C# is supported. 

VB output was never completed implemented. Both `fsproj` and loose F# files are not supported because we currently supports only roslyn-based project systems.

Reply:
Yufei - can you clarify what this means in practice - I am not sure what use the ability to input VB is if it can't be output?

Reply:
You can still build API docs from `vbproj` or loose VB files, its just the output HTML shows the type and member signatures in C# syntax not VB.

Reply:
Just committed a fix to clarify all this.

@KalleOlaviNiemitalo Thanks for bringing this to our attention.  FYI, I am going to produce a FAQ document that covers a bunch of these type of issues and add it to the docs.  My understanding from the above is that 1) Docfx is unlikely to ever support F# (as it is so heavily dependent on Roslyn, and Roslyn is very unlikely to support F#, and 2) VB output could be added but it is probably a stack of work and perhaps not so relevant on the basis that not too many library writers are writing libraries in VB.